helm/templates/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
Does it make sense to deploy this chart without ingress? If not, I would remove this logic.